behat-tests/features/acquia_lift_element_variations.feature

```diff
@@ -1,170 +1,198 @@
     When I click "Webpage elements" in the "modal_content" region
-    And I wait for AJAX to finish
     Then I should not see the modal
-    When I click "#site-name a span" element in the "page_content" region
-    And I wait for AJAX to finish
-    Then I should see the text "<SPAN>" in the "dialog_variation_type" region
+
+    # I add a new variation set.
+    When I click "#page-title" element in the "page_content" region
+    Then I should see the text "<H1>" in the "dialog_variation_type" region
     When I click "Edit text" in the "dialog_variation_type" region
-    And I wait for AJAX to finish
     Then I should not see the variation type dialog
-    And I should see the text "Edit text: <SPAN>" in the "dialog_variation_type_form" region
-    And the "personalize_elements_content" field should contain the site title
-    When I fill in "Test 1" for "personalize_elements_content"
+    And I should see the text "Edit text: <H1>" in the "dialog_variation_type_form" region
+    When I fill in "Test Article Title - Updated 1" for "personalize_elements_content"
     And I fill in "Test variation set" for "title"
-    And I press the "Save" button
-    And I wait for AJAX to finish
+    And I click "#edit-variation-type-submit-form" element in the "dialog_variation_type_form" region
+
+    # I verify my variation set is created.
     Then I should see the message "The variation set has been created." in the messagebox
-    When I wait for AJAX to finish
     Then I should not see the variation type form dialog
-    And I should see the text "Test 1" in the "page_content" region
+    And I should see the text "Test Article Title" in the "page_content" region
     And I should see "1" for the "variation set" count
+
+    # I bring up the "Add variation" interface.
     When I hover over "Variation Sets" in the "lift_tray" region
     Then I should see the text "Test variation set" in the "lift_tray" region
     And I should see the link "Control variation" in the "lift_tray" region
     And I should see the link "Variation #1" in the "lift_tray" region
     And I should see the link "Add variation" in the "lift_tray" region
     And I should see the link "Add variation set" in the "lift_tray" region
+
+    # I add a new variation to to the existing variation set.
     When I click "Add variation" in the "lift_tray" region
-    Then "#site-name a span" element in the "page_content" region should have "acquia-lift-page-variation-item" class
-    And I should see the text "Edit text: <SPAN>" in the "dialog_variation_type_form" region
-    And I should not see the link "Edit selector" visible in the "dialog_variation_type_form" region
-    When I fill in "Test 2" for "personalize_elements_content"
-    And I press the "Save" button
-    And I wait for AJAX to finish
+    Then "#page-title" element in the "page_content" region should have "acquia-lift-page-variation-item" class
```

Comment:
Outside the scope of this... but we should probably move this logic into something more readable.   The acquia-lift-page-variation-item class indicates that the element is highlighted during edit.   Perhaps a todo to remind us next time we come to this.